website_sale_one_step_checkout/README.rst

```diff
@@ -0,0 +1,50 @@
+One Step Checkout
+=================
+
+Provides an All-In-One Checkout for Your Odoo Customer
+
+.. image:: https://github.com/blooparksystems/website/blob/10.0-wip-website_sale_osc/website_sale_one_step_checkout/static/description/osc.png
+
+The One Step Checkout brings different options:
```

Comment:
Update settings area image because we want to provide the country selection and terms and condition also in the default checkout.